spec/features/course/assessment/submission/manually_graded_spec.rb

```diff
@@ -245,29 +182,26 @@
         # Check that student is notified that his submission is graded
         emails = unread_emails_for(student.email).map(&:subject)
         expect(emails).to include('course.mailer.submission_graded_email.subject')
-
-        expect(current_path).
-          to eq(edit_course_assessment_submission_path(course, assessment, submission))
         expect(submission.reload.published?).to be(true)
+        expect(submission.points_awarded).to eq(0)
 
         # Update grade and check if grade fields are updated
-        fill_in find('input.form-control.grade')[:name], with: 1
-        expect(page.all('.submission-statistics-total-grade', text: '1').count).to eq(2)
+        first("input[type='number']").set(1)
```

Comment:
Shall we be a little more specific here, maybe add a selector? Just in case there are other number input fields next time.

Reply:
React does not auto generate selector like rails. We have to manually add them in the code, that means we have to specify quite some classes only for testing.  Currently I only added unless it's very necessary. However, we can make a decision on this and stick with that, which one do you prefer?

Reply:
I'm actually ok for this, but we might be detecting wrong number type inputs next time though, that's why I thought having more specific selectors might be better. 

Quick question - do you think it is possible to 'magically' inject the class field into form fields of redux-form?

Reply:
> I'm actually ok for this, but we might be detecting wrong number type inputs next time though, that's why I thought having more specific selectors might be better.

Understand, but in that case, the tests will most likely fail.  but on the readability wise it will be clear. 

> Quick question - do you think it is possible to 'magically' inject the class field into form fields of redux-form?

Yah I think this is possible to set the class name since the component is defined by ourselves, but have to figure out what to use as class name and in this case it won't help because redux-form is not used for gradings. 

Perhaps I manually add a class `grade` for this case?